### PR TITLE
Check the full_name returned from mozillians.org (fixes #149).

### LINKED ISSUE #149: [traceback] Check the full name returned from mozillians.org
not enough values to unpack (expected 2, got 1)

### PR DESCRIPTION
**Pull request by akatsoulas**
@johngian r?

### CODE REVIEW

File: feedthefox/users/adapters.py [hunk 1]
```diff
@@ -64,7 +64,12 @@ def populate_user(self, request, sociallogin, data):
                 if mozillian[attr].get('privacy') == 'Public':
                     setattr(user, attr, mozillian[attr].get('value'))
             if mozillian['full_name']['privacy'] == 'Public':
-                first_name, last_name = mozillian['full_name']['value'].split(' ', 1)
+                full_name = mozillian['full_name']['value'].split(' ', 1)
+                try:
+                    first_name, last_name = full_name
+                except ValueError:
+                    first_name = full_name[0]
```
Comment (johngian) [inline feedthefox/users/adapters.py:71]: What happens in case there is no `full_name` value?

Comment (akatsoulas) [inline feedthefox/users/adapters.py:71]: full_name is a required field in mozillians.org so there is always at least a character.

Comment (johngian) [inline feedthefox/users/adapters.py:71]: coolio

**Comment by johngian** [comment]
r+

**Pull request merged**